packages/editor/src/client/base.tsx

```diff
@@ -41,12 +43,8 @@ const iconStyles = {
   lineHeight: '56px',
   textAlign: 'center' as const
 }
-// eslint-disable-next-line @typescript-eslint/ban-ts-comment
-// @ts-ignore
-const NavItemLink = routeLink(Nav.Item)
-// eslint-disable-next-line @typescript-eslint/ban-ts-comment
-// @ts-ignore
-const DropdownItemLink = routeLink(Dropdown.Item)
+const NavItemLink = routeLink(Nav.Item as ComponentType<LinkHOCCompatibleProps>)
```

Comment:
Nav.Item and Dropdown.Item contains the interface `StandardProps` that declares `[key: string]: any;`, this is lost further down the line due to inheritance as seem to be an oversight of the package developer, as the interfaces seem to indicate that certain props exists and the rest gets passed down. 

As href & onClick is not being set on them `routeLink` threw an error. Explicitly setting the `LinkHOCCompatibleProps` props, fixes this issue.